driverdisk-disk.ks.in

```diff
@@ -26,7 +26,7 @@ RESULTFILE=$SYSROOT/root/RESULT
 fail() { echo "*** $*" >> $RESULTFILE; }
 
 # check the installer environment
-[ -f /lib/modules/`uname -r`/updates/fake-dd.ko ] || fail "kmod not loaded"
+[ -f /usr/lib/modules/3.0.0/extra/fake-dd.ko ] || fail "kmod not loaded"
```

Comment:
This folder is created by the RPM with driver disk. The RPM is created by the script in ``lib/mkdud.py`` and there is 3.0.0 as kernel version if version is not set. Because the tests are run on boot.iso, we don't know which kernel version is there in time of the script execution.

This can be improved by grabbing logs from Lorax when building this iso or by running the boot.iso in some pre-testing phase but I don't think it's so much valuable for us now. But it could be a nice improvement for the tests in future.